include/marlin/Configuration_A3ides_2209_MINI.h

```diff
@@ -221,23 +221,23 @@
 #if EITHER(PARKING_EXTRUDER, MAGNETIC_PARKING_EXTRUDER)
 
     #define PARKING_EXTRUDER_PARKING_X \
-        { -78, 184 } // X positions for parking the extruders
-    #define PARKING_EXTRUDER_GRAB_DISTANCE 1 // (mm) Distance to move beyond the parking point to grab the extruder
-//#define MANUAL_SOLENOID_CONTROL                   // Manual control of docking solenoids with M380 S / M381
+        { -78, 184 }                         // X positions for parking the extruders
+    #define PARKING_EXTRUDER_GRAB_DISTANCE 1 // (mm) Distance to move beyond the parking point to grab the extruder \
+                                             //#define MANUAL_SOLENOID_CONTROL                   // Manual control of docking solenoids with M380 S / M381
```

Comment:
Format broken there.

Reply:
Thanks for pointing this out.

Fixed (well, I just disabled clang-format for `Configuration_A3ides_2209_MINI.h` and `Configuration_A3ides_2209_MINI_adv.h`).